main.py

```diff
@@ -158,11 +174,21 @@ async def inner_f():
             await current_channel.send(f"Chillin' for {seconds_between_songs} seconds...")
             await asyncio.sleep(seconds_between_songs)
 
-        # After the current song has finished, pop another off the front and continue playing
+        # Pop a song off the front of the queue and play it
         author, filename, local_filepath = current_channel_content.pop(0)
         await current_channel.send(f"**Playing:** {author.mention} - `{filename}`.")
         active_voice_client.play(discord.FFmpegPCMAudio(local_filepath), after=play_next_song)
 
+        # Change the name of the bot to that of the currently playing song.
+        # This allows people to quickly see which song is currently playing.
+        new_nick = f"{author.nick or author.name} - {filename}"
```

Comment:
It might be a good idea to use the singular "ー" character instead of " - " to squeeze out two extra characters for the song name. If you think that looks too strange then feel free to leave it as-is.

Reply:
So I played around with this, and using a `:` as well.

The original (`-`) with spaces:
![Screenshot from 2022-01-09 13-04-34](https://user-images.githubusercontent.com/1342360/148694794-562f62ce-ce5b-4907-b61a-2ca185dfd93b.png)

With an emdash (`ー`) with no spaces:
![Screenshot from 2022-01-09 13-07-05](https://user-images.githubusercontent.com/1342360/148694814-ce64cd24-f604-4419-a380-b44048ca029e.png)

With a colon: 
![Screenshot from 2022-01-09 13-07-54](https://user-images.githubusercontent.com/1342360/148694822-532292b1-2eac-4e30-81bd-f8271cdd28e1.png)

So it seems that Discord actually transforms dashes into emdashes in the UI automatically anyways. Also interesting was that removing the spaces surrounding the emdash only gave us one more character, even though we were eliminating two. Personally I'm not a huge fan of the emdash with no spaces; I think having spaces is more visually pleasing.

I find `:` to be equally visually pleasing, as well as saving more characters. Unfortunately, Discord seems to have a bug here where the nick is cut off short. There should be at least an ellipsis there. Here's another example of the bug:

![Screenshot from 2022-01-09 13-09-48](https://user-images.githubusercontent.com/1342360/148694898-9ce7e144-5de5-470c-abf1-f6f88b120ab4.png)
![Screenshot from 2022-01-09 13-10-01](https://user-images.githubusercontent.com/1342360/148694902-f5c756ec-4046-474a-b22d-81cc1e4afc71.png)

So it was a good idea, but we're limited by what Discord's UI gives us. From the above, I'd opt for keeping the `-` with surrounding spaces.

Reply:
Ah! Actually, there's not a bug with the Discord UI with a colon, it's just that that is 32 characters. So we could do a colon and add an ellipsis to the end, which while wouldn't do a whole lot for the area of the UI I'm screenshotting above, it would allow people to see one more character in the stage/voice channel user listing.

Here's a comparison of ` - ` vs. `: `

![image](https://user-images.githubusercontent.com/1342360/148695349-b3e45f1a-e2f2-4523-8c78-2f38b490fc4a.png)
![image](https://user-images.githubusercontent.com/1342360/148695355-11b4db0f-73eb-49f6-bdef-fd96d72eadf9.png)
![image](https://user-images.githubusercontent.com/1342360/148695393-f34eb621-5679-426b-bb02-4d3fc741cddb.png)
![image](https://user-images.githubusercontent.com/1342360/148695398-0fe84144-aeca-4139-b9cf-837940c268db.png)

Looks OK to me, and grants us one more character.

Reply:
Re-requesting review just to check that these changes make sense before merging.